README.md

```diff
@@ -4,6 +4,8 @@
 
 > A calculator built using react for performing simple mathematical problem.
 
+![calculator](./images/calculator.png)
+
```

Comment:
- [ ] the live link in the readme file is not working, please make sure it works of remove it.